tensorflow/core/kernels/matmul_op_test.cc

```diff
@@ -245,13 +256,16 @@ class FusedMatMulOpTest : public OpsTestBase {
   void VerifyConv2DWithBiasAndActivation(int m, int k, int n, bool transpose_a,
                                          bool transpose_b,
                                          const string& activation) {
+
+    bool use_gpu_device = activation == "Relu" || (this->TValueType == DT_HALF);
+    //VLOG(-1) << "using GPU " << use_gpu_device;
```

Comment:
nit: seems this line is left-over from debugging?

Reply:
yes, this can be removed